library/src/app/components/trade/trade.component.ts

```diff
@@ -70,17 +59,23 @@ export class TradeComponent implements OnInit, OnDestroy {
     amount: new FormControl()
   });
 
-  quote: Quote = {
-    asset: Constants.ASSET,
-    counterAsset: Constants.COUNTER_ASSET,
+  postQuoteBankModel: PostQuoteBankModel = {
+    customer_guid: '',
+    symbol: '',
+    side: PostQuoteBankModel.SideEnum.Buy
+  };
```

Comment:
Is this the model used to communicate with the REST API?

Reply:
Yeah, minus the `receive_amount` or `deliver_amount`.